addons/account/models/account_tax.py

```diff
@@ -194,7 +194,7 @@ class AccountTax(models.Model):
     country_code = fields.Char(related='country_id.code', readonly=True)
     is_used = fields.Boolean(string="Tax used", compute='_compute_is_used')
     repartition_lines_str = fields.Char(string="Repartition Lines", tracking=True, compute='_compute_repartition_lines_str')
-    invoice_legal_notes = fields.Html(string="Legal Notes", help="Legal mentions that have to be printed on the invoices.")
+    invoice_legal_notes = fields.Html(string="Legal Notes", translate=True, help="Legal mentions that have to be printed on the invoices.")
```

Comment:
```suggestion
    invoice_legal_notes = fields.Html(string="Legal Notes", translate=html_translate, help="Legal mentions that have to be printed on the invoices.")
```

Reply:
Is that new ? :exploding_head:

Reply:
Not really :nerd_face: 
https://github.com/odoo/odoo/commit/6c8e53a08705b948833fa58d3c5dc182ebb889c4

Reply:
didn't know that, i will change asap

Reply:
But it's the default if you set `translate=True`, from the same commit...